webauthn4j-core/src/main/java/com/webauthn4j/converter/util/JsonConverter.java

```diff
@@ -39,31 +37,26 @@
  */
 public class JsonConverter implements Serializable {
 
+    public final static JsonConverter INSTANCE = new JsonConverter();
```

Comment:
I don't take singleton pattern for JsonConverter/CborConverter. In some cases, a user of WebAuthn4J would like to register his own class to ObjectMapper owned by one WebAuthnAuthenticationContextValidator instance, but he may not want to register the class to the other WebAuthnAuthenticationContextValidator instance.